.travis.yml

```diff
@@ -20,6 +20,7 @@ matrix:
     - php: nightly
     - php: hhvm
   allow_failures:
+    - php: hhvm
```

Comment:
I would rather remove the job entirely (given that HHVM itself dropped support for PHP interop)

Reply:
I was planning to do a major version bump and drop support for all unmaintained PHP versions after this is merged and we do a small minor version bump.